public/js/views/pay-methods.jsx

```diff
@@ -6,19 +6,30 @@ import CardList from 'components/card-list';
 import { gettext } from 'utils';
 
 
-export default class ManageCards extends Component {
+export default class PaymentMethods extends Component {
 
   static propTypes = {
     closeModal: PropTypes.func.isRequired,
     paymentMethods: PropTypes.array.isRequired,
   };
 
+  renderChild() {
+    if (this.props.paymentMethods && this.props.paymentMethods.length) {
+      return <CardList cards={this.props.paymentMethods} />;
+    }
+    return (<p className="no-cards">
+      {gettext('You don\'t have any cards yet')}
```

Comment:
more grammar nits: 

Not having cards could be a lie. I think it's more like "You haven't added any credit cards yet"